src/layout/Footer.jsx

```diff
@@ -1,7 +1,95 @@
 import React from "react";
+import { Link as GatsbyLink } from "gatsby";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import Link from "@material-ui/core/Link";
+import { Facebook, Twitter, RssFeed } from "@material-ui/icons";
+import Grid from "@material-ui/core/Grid";
+
+/* Defines CSS classes to use for all instances of this component. This is
+  actually a custom React hook. */
+const useStyles = makeStyles((theme) => ({
+  footerLink: {
+    color: theme.palette.text.secondary,
+  },
+}));
 
 const Footer = ({ title }) => {
-  return <>&copy; {title} 2020</>;
+  const classes = useStyles(); //consumes ThemeProvider 
+  return (
+    <>
+      <hr />
+      <Typography
+        variant="h5"
+        color="textSecondary"
+        align="center"
+        noWrap
+      >
+        <Link
+          to="/"
+          component={GatsbyLink}
+          color="inherit"
+        >
+          {title}
+        </Link>
+      </Typography>
+      <Grid 
+        container 
+        direction="row"
+        justify="center" 
+        alignItems="center"
+        spacing={2}
+      >
+        <Grid item>
+          <Link
+            href="https://www.facebook.com/huntercsblog"
+          >
+            <Facebook />
+          </Link>
+        </Grid>
+        <Grid item>
+          <Twitter />
+        </Grid>
+        <Grid item>
+          <RssFeed />
+        </Grid>
+      </Grid>
+      <Grid 
+        container 
+        direction="row"
+        justify="center" 
+        alignItems="center"
+        spacing={2}
+      >
+        <Grid item>
+          <Link 
+            href="mailto:icarus@huntercs.club"
+            className={classes.footerLink}
+          >
+            Contact
+          </Link>
+        </Grid>
+        <Grid item>
+          <Link to="/" className={classes.footerLink}>
+            About Us
+          </Link>
+        </Grid>
+        <Grid item>
+          <Link to="/" className={classes.footerLink}>
+            Write For Us
+          </Link>
+        </Grid>
+        <Grid item>
+          <Link to="/" className={classes.footerLink}>
+            Join the team
+          </Link>
+        </Grid>
+      </Grid>
+      <Typography variant="body1" align="center">
+        &copy; {title} 2020 - {(new Date()).getFullYear()}
```

Comment:
Should we show the `- current year` part for when 2020 is the current year?

Reply:
I thought it looked funny but couldn't think of an expression to conditionally render it in a concise/efficient manner

Reply:
hmm, perhaps declare the `const year = new Date().getFullYear()` outside and then do something simple like `&copy; {title} 2020 {year !== 2020 && `- ${year}`}`